class.js

```diff
@@ -1,20 +1,17 @@
-class Foo
-{
-color='negro';peso   =    34;
-  ver()
-  {
-    console.info(`Color: ${this.color}. Peso: ${this.peso}`
-    );
-// foo = 123;
-/*
-// Prueba
-test() {
-  console.info('Test');
+class Foo {
+  color = 'negro';
+  peso = 34;
+ver() {
+  console.info (`Color: ${this.color}. Peso: ${this.peso}`);
```

Comment:
El contenido de la `function` debe estar indentado "dentro" del método `ver()`.
```suggestion
    console.info (`Color: ${this.color}. Peso: ${this.peso}`);
```